src/action/macos/bootstrap_launchctl_service.rs

```diff
@@ -32,7 +32,7 @@ impl BootstrapLaunchctlService {
         let mut command = Command::new("launchctl");
         command.process_group(0);
         command.arg("print");
-        command.arg(format!("{domain}/{service}"));
+        command.arg("{domain}/{service}");
```

Comment:
.....this seems wrong. Am I wrong?

Reply:
YOu're right, oh my goodness. Wait, how did the tests pass?

Reply:
That's a great question.

We should probably make a `Command` wrapper that will automatically trace the commands being run locally, so we could see if this appeared in the logs...

But I think that's because the command spawned properly (so the `.map_err` into `ActionError::command(...)` never happened), but that it was just an error -- and in that case all we do is mark it uncompleted. The lack of a `format!` would have just made this unconditionally false (and thus always uncompleted).

Reply:
We do have this -- however it also gobbles some errors we don't necessarily want it to (such as in this case):

https://github.com/DeterminateSystems/nix-installer/blob/f9ab680840a486f72b4a8a1176a6e86ba87283f7/src/lib.rs#L94-L104